forge-game/src/main/java/forge/game/card/CardProperty.java

```diff
@@ -173,7 +174,16 @@ public static boolean cardHasProperty(Card card, String property, Player sourceC
             }
         } else if (property.equals("IsTriggerRemembered")) {
             boolean found = false;
-            for (Object o : spellAbility.getTriggerRemembered()) {
+            List<Object> trigRems = spellAbility.getTriggerRemembered();
+            if (trigRems.size() == 0) { // try wrapped
+                SpellAbility sa = (SpellAbility) spellAbility;
+                SpellAbility root = sa.getRootAbility();
+                if (root.isWrapper()) {
+                    root = ((WrappedAbility) root).getWrappedAbility();
+                }
+                trigRems = root.getTriggerRemembered();
+            }
+            for (Object o : trigRems) {
```

Comment:
wouldn't it be better if this part would have been handled by WrappedAbility already?

Reply:
- WrappedAbility actually already has a call for it
- even so, an AbilitySub usually shouldn't have WrappedAbility as parent

so I don't know why it was added